### DISCUSSION TITLE
servo for bolt 1.1

### DISCUSSION THREAD

**Discussion by kyoeun-space** [bitcraze/discussions#729] [Q&A] [answered]
Hello. I'm someone with very little understanding of circuit boards. I want to connect up to four servo motors (SG90) to the bolt 1.1. Do I need the BigQuad deck to connect servos to the bolt 1.1? I have a question because I understand that only the 2.x is supported. I'm waiting for your help. Thank you!

**Comment by knmcguire** [comment]
Hi! The Big quad deck is for the crazyflie 2.x, not the bolt. The combination crazyflie + [bigquad ](https://www.bitcraze.io/products/bigquad-deck/) deck is actualy the same as a bolt (the bq deck existed before the bolt).

Currently @tobbeanton is on a holiday who is more able to help out with this. However, perhaps @matejkarasek can shed some light on this as well, since [the flapper drone](https://store.bitcraze.io/products/flapper-nimble-starter-kit) has servos and uses serves too, but I think you need some more hardware/electronics in between in order to drive it.

**Reply by matejkarasek** [reply, marked as answer]
Hi! You should be able to connect your servo directly to the Bolt, using the cables included with it. Just pay attention to the voltages, the red wire is connected to the battery, so whatever voltage you use to power the Bolt will also go to the servo. In case your servo needs to operate at a different voltage, you could power it externally and only use the white (signal) and black (ground) wires. In that case also interconnect the grounds.
In the firmware, you might need to switch the ESC protocol to PWM as I think it is now DSHOT by default. This can be done through Kconfig.

However, bigquad deck should also work, again you might need to do some minor tweaks in the firmware to get it working.

**Reply by matejkarasek** [reply]
Forgot to add - in both cases you will need to do some soldering, with the Bigquad that would likely be less

**Reply by kyoeun-space** [reply]
Thank you for providing detailed information! Based on your replies, I'm plannig to create what I have imagined, soon. I hope I can show you a successful drone. Thank you all!!
